code/procs/antagonist_procs.dm

```diff
@@ -393,7 +356,7 @@ var/list/roles_to_prefs = list(
 			added_text = "spy"
 		if(ROLE_HEAD_REV)
 			added_text = "rev"
-		if(ROLE_NUKEOP)
+		if((ROLE_NUKEOP || ROLE_NUKEOP_COMMANDER))
```

Comment:
```suggestion
		if(ROLE_NUKEOP || ROLE_NUKEOP_COMMANDER)
```

Reply:
The brackets indicate to the debugger that the `or` is not an error when used in a switch-case branch, which will otherwise issue a warning message.

Reply:
wait yeah it should be warning on this lol, this isn't how you do switch statements

Reply:
```suggestion
		if(ROLE_NUKEOP, ROLE_NUKEOP_COMMANDER)
```

Reply:
There's another switch statement similar to this in `stat_logging.dm`; I'll correct the syntax then push the changes.